### PR TITLE
add option ipv4 support

### PR DESCRIPTION
**Pull request by huankong233**
this may fix

- [#33](https://github.com/huankong233/94list-laravel/issues/33)
- [#3191](https://github.com/guzzle/guzzle/issues/3191)

I convert the ipv4 part into hex and follow the rule set the `prefix` and `url` variable

### CODE REVIEW

**Comment by huankong233** [comment]
time to merge right? XD

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
I can't merge this without robust test coverage. Please feel free to send a new PR, if you can add this.

**Pull request closed without merging**
